devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -1065,6 +1074,7 @@ bool YarpRobotLoggerDevice::hasSubstring(const std::string& str,
 
 void YarpRobotLoggerDevice::lookForNewLogs()
 {
+    std::cout << "Looking for new logs" << std::endl;
```

Comment:
Leftover?

Reply:
Fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)